code/modules/food_and_drinks/kitchen_machinery/smartfridge.dm

```diff
@@ -239,6 +239,24 @@
 	icon_off = "drying_rack"
 	var/drying = 0
 
+/obj/machinery/smartfridge/drying_rack/New()
+	..()
+	if(component_parts && component_parts.len)
+		component_parts.Cut()
+	component_parts = null
+
+/obj/machinery/smartfridge/drying_rack/on_deconstruction()
+	new /obj/item/stack/sheet/mineral/wood(loc, 10)
```

Comment:
no advantage, that is how the code is intended to be used, so I just followed it